### PR TITLE
homebrew config using goreleaser

### LINKED ISSUE #25: add `pctl` cli as package to Homebrew package manager
### Briefly describe the feature

- add this CLI tool in Homebrew package manager which is widely used by Mac users. Adding to their repo will give more visibility to the wide userbase. [brew.sh](https://brew.sh/)

### What problem does this feature solve? Please link any relevant documentation or Issues

- having tool in package manager like `brew` will make installation process easy

### _(optional)_ What is your current workaround?

- NONE

### PR DESCRIPTION
**Pull request by rustiever**
### What does this PR change?

- now we can install `pctl` using `homebrew`. To install pls run following command
```sh
brew tap rustiever/homebrew-rustiever # need to change to paralus oragnization
brew info pctl
brew install pctl
```

### Does the PR depend on any other PRs or Issues? If yes, please list them.

- aims to fix #25 

### Checklist

I confirm, that I have...

- [x] Read and followed the contributing guide in `CONTRIBUTING.md`
- [ ] Added tests for this PR
- [ ] Formatted the code using `go fmt` (if applicable)
- [ ] Updated [documentation on the Paralus docs site](https://github.com/paralus/website/blob/main/docs) (if applicable)
- [ ] Updated `CHANGELOG.md`

### CODE REVIEW

**Comment by rustiever** [comment]
hi @niravparikh05 when we specify the same repository to generate brew installation script its not working properly, when we have a seperate repo in this following syntax `homebrew-<name>` eg: homebrew-rustiever this is working.

**Comment by sandeep540** [comment]
Screenshot 2022-11-01 at 11 48 53 AM
Screenshot 2022-11-01 at 11 49 17 AM


First 2 Commands work , **but install fails**

```
brew tap rustiever/homebrew-rustiever # need to change to paralus oragnization
brew info pctl
```

I have latest MacOS 13 - Ventura, updated a week back, this is not working on it, need to check in other mac with older version

**Comment by rustiever** [comment]
`brew install pctl` works in `Mac OS Monetery 12`. i think the issue is with binary. `goreleaser` building the binary with older version macOS i guess. @sandeep540 can you check once downloading the binary from release page of github. if it is not working then most likely issues is with `goreleaser`.
cc: @niravparikh05 @meain

**Comment by sandeep540** [comment]
Screenshot 2022-11-01 at 10 56 09 PM

Binary works

**Comment by meain** [comment]
@sandeep540 The install failing is not because of an issue with pctl, but rather a missing xcode installation. You can install the required deps by running `sudo xcode-select --install`. This installs the xcode command line tools.

**Comment by sandeep540** [comment]
@meain tried removing and installing twice, no luck, same issue

**Comment by meain** [comment]
You'll have to separately install the xcode cli utils before you install pctl.

**Comment by sandeep540** [comment]
Screenshot 2022-11-02 at 4 00 36 PM
Just received a software update for xcode, after the update it works

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
         run: go test -v ./...
 
       - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@0110a4acb73fda6a2c401b6d1cb70a11d5b0fadf
+        uses: goreleaser/goreleaser-action@v2
```
Comment (meain) [inline .github/workflows/release.yaml:40]: I think this was explicitly set by @akshay196 to be a sha

Comment (rustiever) [reply .github/workflows/release.yaml:40]: any specific reason @meain @akshay196-rafay?? but now `v2` works properly.

Comment (akshay196) [reply .github/workflows/release.yaml:40]: Reference from https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions:
> Pinning an action to a full length commit SHA is currently the only way to use an action as an immutable release. Pinning to a particular SHA helps mitigate the risk of a bad actor adding a backdoor to the action's repository, as they would need to generate a SHA-1 collision for a valid Git object payload.

@rustiever You can use SHA of a commit from v2 release.

Comment (rustiever) [reply .github/workflows/release.yaml]: updated @akshay196

**Review by niravparikh05 — APPROVED** [review]
LGTM

**Pull request merged**